### ISSUE TITLE
Got 400 error 'unable to parse' when tag value contains Chinese

### ISSUE THREAD

**Issue by inetfuture** [node-influx/node-influx#244]
```js
influx.writePoints([{
  measurement: 'hourlyReservations',
  tags: { restaurant: '中文' },
  fields: { count: 10 }
}]);
```

`Error: A 400 Bad Request error occurred: {"error":"unable to parse 'hourlyReservations,restaurant=中文 coun': invalid field format"}`

I'm using node-influx 5.0.3, influxdb 1.0.2, node.js 6.2.1

**Comment by inetfuture** [comment]
Also not work when filed value contains Chinese, `A 400 Bad Request error occurred: {"error":"unable to parse \'hourlyReservations count=1,restaurant=\\"中\': unbalanced quotes"}\n`

**Comment by inetfuture** [comment]
Double quoting the value does not work neither:

```js
influx.writePoints([{
  measurement: 'hourlyReservations',
  tags: { restaurant: '"中文"' },
  fields: { count: 10 }
}]);
```

But it works like this:

`curl -i -XPOST 'http://localhost:8086/write?db=aaron' --data-binary 'hourlyReservations,restaurant="中文" count=10'`

**Comment by inetfuture** [comment]
![screen shot 2016-12-01 at 17 21 35](https://cloud.githubusercontent.com/assets/1363189/20788189/b264daba-b7ea-11e6-8177-f173971114c1.png)

Looks like this request body is incomplete?

**Comment by inetfuture** [comment]
Would you please relase this fix to npm?

**Comment by connor4312** [comment]
It's been published in 5.0.4
